flink/v1.20/flink-runtime/NOTICE

```diff
@@ -7,10 +6,48 @@ The Apache Software Foundation (http://www.apache.org/).
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Apache ORC with the following in its NOTICE file:
+This binary artifact includes Apache Avro 1.12.0 with the following in its NOTICE file:
+
+| Apache Avro
+| Copyright 2010 The Apache Software Foundation
+|
+| This product includes software developed at
+| The Apache Software Foundation (http://www.apache.org/).
+|
+| C JSON parsing provided by Jansson and
+| written by Petri Lehtinen. The original software is
+| available from http://www.digip.org/jansson/.
+
+--------------------------------------------------------------------------------
+
+This binary artifact includes Apache Commons Codec 1.9 with the following in its NOTICE file:
+
+| Apache Commons Codec
+| Copyright 2002-2013 The Apache Software Foundation
+|
+| This product includes software developed at
+| The Apache Software Foundation (http://www.apache.org/).
+|
+| src/test/org/apache/commons/codec/language/DoubleMetaphoneTest.java
+| contains test data from http://aspell.net/test/orig/batch0.tab.
+| Copyright (C) 2002 Kevin Atkinson (kevina@gnu.org)
```

Comment:
I don't think we ship this

Reply:
It's the `NOTICE` from commons-codec.

Reply:
But shouldn't we include stuff in the notice when is it strickly required? This is pointing at a class in `test/org/apache/commons/`